packages/app/src/pages/Runner.vue

```diff
@@ -1,15 +1,14 @@
 <template>
-  <div v-if="query.data.value?.app?.activeProject">
-    <SpecRunnerContainer
-      :gql="query.data.value.app"
-    />
-  </div>
+  <SpecRunnerContainer
```

Comment:
Could we make sure that all the "index" pages have the same format?

I would prefer for this one to be called `SpecRunnerPage.vue` or `RunnerPage.vue`.
We could also change `RunsPage.vue` into `RunsContainer.vue`.

Up to you.